assets/src/edit-story/components/panels/document/status/status.js

```diff
@@ -128,31 +145,40 @@ function StatusPanel() {
   return (
     <SimplePanel
       name="status"
-      title={__('Status & Visibility', 'web-stories')}
+      title={__('Status and visibility', 'web-stories')}
       collapsedByDefault={false}
     >
       <>
+        <Row>
+          <HelperText>
+            {__('Set the current status of your story to', 'web-stories')}
+          </HelperText>
+        </Row>
         <Row>
           <RadioGroup
+            groupLabel="Visibility"
+            name="radio-group-visibility"
             options={visibilityOptions}
             onChange={handleChangeVisibility}
             value={getStatusValue(status)}
           />
         </Row>
         {passwordProtected === status && (
```

Comment:
This input is still hidden. Does this need to be visible now? Not sure if the password protection works in the editor yet

Reply:
As mentioned in https://github.com/google/web-stories-wp/pull/6730#discussion_r594166257 we can keep it hidden for now. #2335 will follow-up on this.